assets/css/custom.scss

```
@@ -3908,3 +3908,1394 @@ font-size: 1.55rem;
     }
   }
 }
+
+@keyframes csHeroAsideFloat {
```

Stylelint (17.12.0)

[error] Expected keyframe name "csHeroAsideFloat" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)